ci/libvcx.dockerfile

```diff
@@ -58,7 +107,7 @@ COPY --from=BASE --chown=indy /usr/lib/libindy.so /usr/lib/libindy.so
 COPY --from=BASE --chown=indy /usr/lib/libvcx.so /usr/lib/libvcx.so
 COPY --from=BASE --chown=indy /usr/lib/libnullpay.so /usr/lib/libnullpay.so
 COPY --from=BASE --chown=indy /usr/lib/libindystrgpostgres.so /usr/lib/libindystrgpostgres.so
-
+#
 COPY --from=BASE --chown=indy /home/indy/libvcx ./libvcx
 COPY --from=BASE --chown=indy /home/indy/wrappers/node ./wrappers/node
```

Comment:
This is potentially unncessary and can further reduce the image size

Reply:
But we need to test and publish node stuff. I was thinking of splitting the dockerfile into having
- libvcx without node, rust, without wrappers
- libvcx "testing image" built on top of the one above, used for testing in CI

But again, let's leave that for later. The small changes in this PR reduces image size since already by couple GB